generated/nidaqmx/_task_modules/channels/channel.py

```diff
@@ -56,7 +56,7 @@ def __eq__(self, other):
         return False
 
     def __hash__(self):
-        return hash((self._handle.value, frozenset(self.channel_names)))
+        return self._interpreter.hash_task_sequence((self._handle, frozenset(self.channel_names)))
```

Comment:
The interpreter should only be responsible for hashing task handles, not sequences.
```suggestion
        return self._interpreter.hash_task_handle(self._handle) ^ hash(frozenset(self.channel_names)))
```